packages/test/test-end-to-end-tests/package.json

```diff
@@ -98,6 +98,7 @@
     "@fluidframework/test-utils": "^0.46.0",
     "@fluidframework/test-version-utils": "^0.46.0",
     "@fluidframework/undo-redo": "^0.46.0",
+    "compare-versions": "^3.6.0",
```

Comment:
Curious why we don't just use "semver" package that is used elsewhere so that we have one less package to install?
But if we want to keep this, wouldn't the global lerna-package-lock.json needs to be updated?

Reply:
I was looking for "semver-compare" package which I found on npm and as suggested by you and I didn't find it in our repo. So went with another package. Now I see that we do use "semver" package already so just using that. 

Good catch about the lerna. I had the local change but missed to push it with the changes.